### PR TITLE
change branch of spirv_cross

### LINKED ISSUE #25991: Bug: attempting to run Servo leads to a panic
After building Servo, I ran `./mach run --release tests/html/about-mozilla.html`. Here's the error that followed:

```
attempted to leave type `std::mem::ManuallyDrop<xlib_xcb::Xlib_xcb>` uninitialized, which is invalid (thread main, at /rustc/45ebd5808afd3df7ba842797c0fcd4447ddf30fb/src/libcore/mem/mod.rs:536)
stack backtrace:
   0: servo::backtrace::print
   1: servo::main::{{closure}}
   2: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:513
   3: rust_begin_unwind
             at src/libstd/panicking.rs:417
   4: core::panicking::panic_fmt
             at src/libcore/panicking.rs:111
   5: core::panicking::panic
             at src/libcore/panicking.rs:54
   6: x11_dl::xlib_xcb::Xlib_xcb::open
   7: winit::platform::platform::x11::xdisplay::XConnection::new
   8: std::sync::once::Once::call_once::{{closure}}
   9: std::sync::once::Once::call_inner
             at src/libstd/sync/once.rs:416
  10: winit::platform::platform::EventsLoop::new_x11
  11: winit::platform::platform::EventsLoop::new
  12: winit::EventsLoop::new
  13: servo::events_loop::EventsLoop::new
  14: servo::app::App::run
  15: servo::main
  16: std::rt::lang_start::{{closure}}
  17: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:52
      std::panicking::try::do_call
             at src/libstd/panicking.rs:329
      std::panicking::try::do_try
             at src/libstd/panicking.rs:285
      std::panicking::try
             at src/libstd/panicking.rs:274
      std::panic::catch_unwind
             at src/libstd/panic.rs:394
      std::rt::lang_start_internal
             at src/libstd/rt.rs:51
  18: main
  19: __libc_start_main
  20: _start
Servo exited with return value 101
```

### PR DESCRIPTION
**Pull request by pshaughn**
Hopefully fixes #25991

### CODE REVIEW

**Comment by pshaughn** [comment]
It does not yet do so... @kvark could you please put the commit were were locked at on the tree?

**Comment by kvark** [comment]
bors try

**Comment by jdm** [comment]
@bors-servo retry p=10
* #22212

File: Cargo.toml [hunk 1]
```diff
@@ -30,5 +30,5 @@ opt-level = 3
 mio = { git = "https://github.com/servo/mio.git", branch = "servo" }
 # https://github.com/retep998/winapi-rs/pull/816
 winapi = { git = "https://github.com/servo/winapi-rs", branch = "patch-1" }
-spirv_cross = { git = "https://github.com/kvark/spirv_cross", branch = "wgpu" }
+spirv_cross = { git = "https://github.com/kvark/spirv_cross", branch = "wgpu-servo" }
```
Comment (SimonSapin) [inline Cargo.toml]: Please also change the URL to https://github.com/servo/spirv_cross, which I’ve just forked from kvark/spirv_cross.

**Review by Manishearth — APPROVED** [review]
r+ with simon's change

**Pull request merged**